pkg/kubelet/cm/cgroupManager_mock.go

```diff
@@ -0,0 +1,45 @@
+// +build linux
```

Comment:
don't think we use mixed cases in the file names. s/cgroupManager_mock.go/cgroup_manager_mock.go.

Also, mocks are only used for testing and to avoid it being compiled into the binary. You should create a "testing" subpackage (e.g., see cadvisor/testing/) and move this file there.